changelogs/fragments/1916-add-version-sort-filter.yml

```
@@ -0,0 +1,3 @@
+add plugin.filter:
+  - name: version_sort
+    description: sort a list according to version order instead of pure alphabetical one
```

```suggestion
    description: Sort a list according to version order instead of pure alphabetical one
```
(This is similar to `short_description` in a module or plugin, there we always start upper case and have no period at the end.)